data/themes/darktable.css

```diff
@@ -638,7 +638,7 @@ overshoot.right
 #lib-plugin-ui .section-expander,
 #iop-plugin-ui .section-expander
 {
-  margin: 0.35em 0;
+  margin: 0.5em 0 0.35em 0;
```

Comment:
And then we don't need this.

Reply:
I still prefer a little more top margin as well

Reply:
0.35 or 0.5em are both good for me, but no more yes.

Reply:
Agreed - I wouldn't want to make it any bigger than 0.5.